src/core/exception.d

```diff
@@ -433,8 +433,15 @@ deprecated void setAssertHandler( AssertHandler h ) @trusted nothrow @nogc
  */
 extern (C) void onAssertError( string file = __FILE__, size_t line = __LINE__ ) nothrow
 {
-    if( _assertHandler is null )
-        throw new AssertError( file, line );
+    import core.stdc.string : memcpy;
+    char[1024] buffer;
```

Comment:
Not to mention the alignment issue - I don't think there is guarantee that char[x] array is properly aligned.